Library/Contributions/example-formula.rb

```diff
@@ -30,8 +30,8 @@ class ExampleFormula < Formula
   version "1.2-final"
 
   # For integrity and security, we verify the hash (`openssl dgst -sha1 <FILE>`)
-  # You may also use sha256 if the software uses sha256 on their homepage.
-  # Leave it empty at first and `brew install` will tell you the expected.
+  # You may also use sha256 if the software uses sha256 on their homepage. Do not use MD5.
```

Comment:
Solid point. All caps doesn't match the rest of the sha language in the sentence. Fixed.